### PR TITLE
[iOS] Instructions for testing on physical devices

### PR DESCRIPTION
**Pull request by jahorton**
Modifies README.md with instructions on physical device testing for the iOS app.

### CODE REVIEW

**Comment by darcywong00** [comment]
Is there a minimum iOS version limitation for the physical device?  If so, include that in the "Minimum iOS Requirements" section.

File: ios/README.md [hunk 1]
```diff
@@ -5,6 +5,21 @@ Xcode
...
 
 To run the app within a simulator after build.sh is completed, "Run" the Keyman scheme/target within Xcode.
+
+### Running the app on a physical device
+In order to test any code changes on a physical device, Apple requires that an app be code-signed.  The quickest way to handle this for personal testing:
+
+- Open Xcode and the settings for the Keyman target.
+- Establish personal code-signing permissions.
+  - Go to the **General** tab and enable "Automatically manage signing" and select your "Personal Team."
+  - Afterward, swap to the **Capabilities** tab and *disable* "App Groups."
```
Comment (mcdurdin) [inline ios/README.md]: If you disable App Groups, will you be able to install keyboards and make them available to the system keyboard?

Comment (jahorton) [reply ios/README.md]: Good catch.  You can add them to the app, but not the system keyboard.

Comment (jahorton) [reply ios/README.md]: Interestingly, the "Get Started" item will not report the "install languages" op as successful when languages are downloaded if the system keyboard can't access them.

File: ios/README.md [hunk 2]
```diff
@@ -5,6 +5,24 @@ Xcode
...
+### Running the app on a physical device
+In order to test any code changes on a physical device, Apple requires that an app be code-signed.  The quickest way to handle this for personal testing:
+
+- Open Xcode and the settings for the Keyman target.
+- Establish personal code-signing permissions.
+  - Go to the **General** tab and enable "Automatically manage signing" and select your "Personal Team."
+	- Change the "Bundle Identifier" for both the Keyman and SWKeyboard targets to something personal and unique.
+  - Afterward, swap to the **Capabilities** tab, remove the existing "App Groups", and define your own unique group for the Keyman target.  
+	- Then, make the "App Group" settings of SWKeyboard match those of Keyman.
+	- Perform a find and replace for the group identifier "group.KM41" and with your selected unique group name.
```
Comment (mcdurdin) [inline ios/README.md]: `KM4I` not `KM41`

File: ios/README.md [hunk 3]
```diff
@@ -5,6 +5,24 @@ Xcode
...
+  - Correct any further error messages Xcode presents you with.
+  - **Note**:  All changes for this step (code-signing permissions, app groups, bundle identifier, etc) must be reverted to the original settings for submitted PRs.
+- Under the **Product > Destination** Xcode menu, select the device you wish to use for testing.
+- Attempt to "Run" Keyman on the device.  The operation should fail, but this step is necessary to continue.
+- Now, on the device itself, go to **Settings > General > Profiles & Device Management** and give your developer account trusted permissions.
+
+You should then be freely able to test the app on your personal development device so long as it is running iOS 8 or higher.
```
Comment (darcywong00) [inline ios/README.md]: Yeah, but maybe move the "iOS8 or higher" to the top of the readme

##  Minimum iOS Requirements

**Pull request merged**